### ISSUE TITLE
TDS should clear out S3 cache

### ISSUE THREAD

**Issue by dopplershift** [Unidata/thredds#579]
Right now S3Objects can grow unbounded, at least for as long as the server is running. For the TDS running on top of the S3 NEXRAD archive, this is pretty damn big. Right now I'm running a cronjob to clear it out periodically, but TDS really should manage that.

**Comment by cwardgar** [comment]
Good call. [This](https://github.com/Unidata/thredds/blob/master/cdm/src/main/java/thredds/crawlabledataset/s3/CrawlableDatasetAmazonS3.java#L96) is the relevant method.
